### DISCUSSION TITLE
High CPU usage while using on production using nginx

### DISCUSSION THREAD

**Discussion by ashutoshsingh25** [fastapi/fastapi#8727] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Description=Gunicorn instance to serve myproject
After=network.target

[Service]
User=user
Group=www-data
WorkingDirectory=/home/My_Api
Environment="PATH=/home/My_Api/myproject/myprojectenv/bin"
ExecStart=/home/My_Api/myproject/myprojectenv/bin/gunicorn -w 17 -k uvicorn.workers.UvicornWorker asgi:app --bind unix:app.sock


[Install]
WantedBy=multi-user.target
```

### Description

I have planned my migrate one of my API from flask to fastapi.
While using it in production on a 8 vCPUs server, it start running fine for first 10 mins and after that one or two CPU usage reach to 100%.
I am using gunicorn, uvicorn and nginx to server the same on production.

I have developed the same application in the flask with Nginx and gunicorn and its running perfectly fine on same server for same number of requests.

Sharing screenshot for CPU usage
![image](https://user-images.githubusercontent.com/40763103/148233677-d4e780d7-25f1-4140-b315-1e70d1e06302.png)

### Operating System

Linux

### Operating System Details

Ubuntu 20.1

### FastAPI Version

0.70.1

### Python Version

3.8.1

**Comment by abodacs** [comment]
@ashutoshsingh25 
This is the same issue that happened for clubhouse 
https://blog.clubhouse.com/reining-in-the-thundering-herd-with-django-and-gunicorn/
